_posts/en/newsletters/2021-06-16-newsletter.md

```diff
@@ -0,0 +1,191 @@
+---
+title: 'Bitcoin Optech Newsletter #153'
+permalink: /en/newsletters/2021/06/16/
+name: 2021-06-16-newsletter
+slug: 2021-06-16-newsletter
+type: newsletter
+layout: newsletter
+lang: en
+---
+This week's newsletter celebrates the lock in of the taproot soft fork,
+describes a draft BIP for improving transaction
+privacy by varying the fields used to implement anti fee sniping, and
+features an article about the challenges of mixing transaction
+replacement with payment batching.  Also included are our regular
+sections with announcements of new software releases and release
+candidates, plus notable changes to popular Bitcoin infrastructure
+software.
+
+## News
+
+- **🟩  Taproot locked in:** the [taproot][topic taproot] soft fork and
+  related changes specified in BIPs [340][bip340], [341][bip341], and
+  [342][bip342] were locked in by signaling miners last weekend.
+  Taproot will be safe to use after block 709,632, which is expected in
+  early or mid November.  The delay gives time for users to upgrade
+  their nodes to a release (such as Bitcoin Core 0.21.1) that will
+  enforce taproot's rules, ensuring that funds received to taproot
+  scripts after block 709,632 are safe even if there's a problem with
+  miners.
+
+    Developers are encouraged to start [implementing taproot][taproot
+    uses] in their software during the next several months so they can
+    be ready to take advantage of greater efficiency, privacy, and
+    fungibility as soon as the activation is complete.
```

Comment:
"start implementing [...] during the next months [...] to be ready" feels like a bit too hedged to me. 

I'd either `s/start implementing/implement/` or `s/during the next several months//`